net/host.go

```diff
@@ -131,9 +157,10 @@ func (h *Host) CloseProtocolStream(id types.Hash) {
 	_ = swap.stream.Close()
 }
 
-// Advertise advertises in the DHT.
-func (h *Host) Advertise(strs []string) {
-	h.h.Advertise(strs)
+// RefreshNamespaces forces an immediate update to the advertised namespaces in
+// the DHT
+func (h *Host) RefreshNamespaces() {
+	h.h.RefreshNamespaces()
```

Comment:
not necessarily, it only adds new "namespaces" immediately, but doesn't purge ones if you decide to not readvertise them. I'd call this `Advertise`

Reply:
The comment was updated too.